src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -115,15 +116,18 @@ public GetAccountResponse getAccount(String accountId, String bearerToken) throw
     }
 
     @Override
-    public GetBalanceResponse getBalance(final String accountId, final String bearerToken) throws XpringException {
+    public AccountBalanceResponse getBalance(final String accountId, final String bearerToken) throws XpringException {
```

Comment:
This strikes me as an awkward name, why not `AccountBalance`. `Response` seems extra and not conveying useful information.

Reply:
I don't feel strongly that it should be called `AccountBalanceResponse`.  I basically copied the Hermes REST classes, and that is what we called it in Hermes. But since this is more public facing, I'm happy to rename to `AccountBalance`